include/adm/utilities/object_creation.hpp

```diff
@@ -66,6 +66,30 @@ namespace adm {
   ADM_EXPORT SimpleObjectHolder addSimpleObjectTo(
       std::shared_ptr<Document> document, const std::string& name);
 
+  /**
+   * @brief Create `AudioObject` hierarchy for single
+   * `TypeDefinition::OBJECTS`-type element
+   *
+   * Creates an `AudioObject` including referenced `AudioPackFormat` and
+   * `AudioChannelFormat` of type `TypeDefinition::OBJECTS`, as well an
+   * `AudioTrackUid`. This creates a shorter structure than 
+   * `createSimpleObject` as supported by 2076-2.
```

Comment:
I'd prefer this to say what the structure is -- "short" is good as a name, but i'm not sure it's clear outside of the context of this PR. How about something like this instead of the last sentence:

> The audioTrackUID references the audioChannelFormat directly, without an audioTrackFormat and audioStreamFormat, as supported by BS.2076-2.